source/index.html

```diff
@@ -1,12 +1,175 @@
 <!DOCTYPE html>
-<html lang="ru">
-  <head>
-    <meta charset="utf-8">
-    <title>HTML Academy: Седона</title>
-  </head>
-  <body>
-
-    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/adaptive">HTML и CSS. Адаптивная вёрстка и автоматизация</a>».</p>
-
-  </body>
+<html class="page" lang="ru">
+<head>
+  <meta charset="UTF-8">
+  <title>Седона</title>
+</head>
+<body class="page__body">
+  <header class="page-header">
+    <a class="page-header__logo" href="index.html">
+      <img src="img/mobile/logotype-mobile.svg" width="102" height="84" alt="Седона">
+    </a>
+      <nav class="main-nav">
+        <button class="main-nav__toggle" type="button">
+          <span class="visually-hidden">Открыть меню</span>
+        </button>
+        <div class="main-nav__wrapper">
+          <ul class="main-nav__list">
+            <li class="main-nav__item">
+              <a class="main-nav__link main-nav__link--current" href="index.html">
+                Главная
+              </a>
+            </li>
+            <li class="main-nav__item">
+              <a class="main-nav__link" href="catalog.html">
+                Фото и видео
+              </a>
+            </li>
+            <li class="main-nav__item">
+              <a class="main-nav__link" href="form.html">
+                Форма отзыва
+              </a>
+            </li>
+            <li class="main-nav__item">
+              <a class="main-nav__link" href="https://htmlacademy.ru"  target="_blank">
+                Html academy
+              </a>
+            </li>
+          </ul>
+        </div>
+      </nav>
+  </header>
+  <main class="page-main">
+    <h1 class="visually-hidden">Поиск гостиниц в небольшом городке "Седона"</h1>
+    <section class="intro">
+      <h2 class="visually-hidden">Добро пожаловать в Седону!</h2>
+      <div class="intro-wrapper">
+        <img src="img/mobile/index/intro-text.svg" width="282" height="214" alt="Добро пожаловать в Седону">
+      </div>
+    </section>
+    <section class="advantages">
+      <div class="advantages__wrapper">
+        <header class="advantages__header">
+          <h2 class="advantages__title">Седона — небольшой городок в Аризоне,заслуживающий большего!</h2>
+          <p class="advantages__intro">
+            Рассмотрим 5 причин, по которым Седона круче, чем гранд каньон!
+          </p>
+        </header>
+        <ol class="advantages__list">
+          <li class="advantage advantage--color">
```

Comment:
нет смысла делать лишку бем-элементом. Если мы вырвем этот блок из контекста, он утеряет свой смысл, так как он зависим от того, где он находтся. Сделай бем-элементом: 
```suggestion
        <ol class="advantages__list">
          <li class="advantages__item advantages__item--color">
```